examples/undocumented/python/evaluation_cross_validation_mkl_weight_storage.py

```diff
@@ -71,11 +72,11 @@ def evaluation_cross_validation_mkl_weight_storage(traindat=traindat, label_trai
     # print mkl weights
     weights = []
     for obs_index in range(mkl_storage.get_num_observations()):
-        obs = mkl_storage.get_observation(obs_index)
-        for fold_index in range(obs.get_num_folds()):
-            fold = obs.get_fold(fold_index)
-            machine = MKLClassification.obtain_from_generic(fold.get_trained_machine())
-            w = machine.get_kernel().get_subkernel_weights()
+        obs = mkl_storage.get_observation(obs_index).get("cross_validation_run")
+        for fold_index in range(obs.get("num_folds")):
+            fold = obs.get("folds", fold_index)
+            machine = as_machine(fold.get("trained_machine"))
+            w = as_kernel(machine.get("kernel")).get_subkernel_weights()
```

Comment:
`get_subkernel_weights` is not part of `CKernel` or?
Also, I don't think you need to do the cast here, just can do
`obs.get("folds", fold_index).get("trained_machine").get("subkernel_weights_or_whatever_the_name_is")`